builder.go

```diff
@@ -42,14 +43,17 @@ func NewBuilder(maxSquareSize int, subtreeRootThreshold int, txs ...[]byte) (*Bu
 		maxSquareSize:        maxSquareSize,
 		subtreeRootThreshold: subtreeRootThreshold,
 		Blobs:                make([]*Element, 0),
-		Pfbs:                 make([]*share.IndexWrapper, 0),
+		Pfbs:                 make([]*v1.IndexWrapper, 0),
 		Txs:                  make([][]byte, 0),
 		TxCounter:            share.NewCompactShareCounter(),
 		PfbCounter:           share.NewCompactShareCounter(),
 	}
 	seenFirstBlobTx := false
 	for idx, tx := range txs {
-		blobTx, isBlobTx := share.UnmarshalBlobTx(tx)
+		blobTx, isBlobTx, err := share.UnmarshalBlobTx(tx)
+		if err != nil && isBlobTx {
```

Comment:
what if `err !=nil` but `isBlobTx == false`, where does the unmarshalling error get returned?